waterloop-site/src/theme/global.scss

```diff
@@ -566,41 +574,34 @@ h1 {
 
 .landingWaterloop {
   /* type img */
-  width: 700px;
+  max-width: 700px;
+  width: 90vw;
   height: auto;
   -webkit-transition: 0.2s ease-in-out;
   transition: 0.2s ease-in-out;
 }
 
-@media (max-width: 420px) {
+@media (max-width: 500px) {
+  .landingAligner {
+    min-height: 100vw;
+  }
+
+  .landingContainer {
+    background-image: none!important;
```

Comment:
is there some way to do this without `!important`?  `!important` is an anti-pattern and should be avoided whenever possible.